css/libs/vendors/codemirror/base16-dark.css

```diff
@@ -8,11 +8,11 @@
 
 */
 
-.cm-s-base16-dark.CodeMirror { background: #151515; color: #e0e0e0; }
+.cm-s-base16-dark.CodeMirror { background: #1e1e1e; color: #e0e0e0; }
```

Comment:
We should not change the vendors files directly but override their css inside our `styles.css`. Otherwise, the day we need to upgrade them, we will have to make a diff and patch the new version.